packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -38,23 +39,21 @@ class Group extends Component {
         }
     }
 
-    mapItems = ({ groupValue, onSelect, groupLabel, groupId, type, items, ...group }, groupKey) => {
+    mapItems = ({ groupValue, onSelect, groupLabel, groupId, type, variant, items, ...group }, groupKey) => {
         const { onFilter } = this.props;
         const { filterBy } = this.state;
         let input;
-
         try {
             input = new RegExp(filterBy, 'i');
         } catch (err) {
             input = new RegExp(filterBy.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'i');
         }
 
-        return items.filter(item => onFilter || (
+        return items.filter(item =>
```

Comment:
Is there a reason to remove the `onFilter` function?

Reply:
oh I was trying something and missed not writing it back then, thanks for catching that!